packages/samples/atomic-react/public/app.css

```diff
@@ -0,0 +1,168 @@
+atomic-color-facet[field='cat_color']::part(value-Black) {
+  background-color: #222222;
+}
+
+atomic-color-facet[field='cat_color']::part(value-White) {
+  background-color: white;
+  border: 1px solid var(--atomic-neutral);
+}
+
+atomic-color-facet[field='cat_color']::part(value-Stainless) {
+  background-image: linear-gradient(315deg, #d3d3d3 0%, #57606f 74%);
+}
+
+atomic-color-facet[field='cat_color']::part(value-Silver) {
+  background-color: silver;
+  border: 1px solid var(--atomic-neutral);
+}
+
+atomic-color-facet[field='cat_color']::part(value-Gray) {
+  background-color: gray;
+  border: 1px solid var(--atomic-neutral);
+}
+
+atomic-color-facet[field='cat_color']::part(value-Blue) {
+  background-color: rgb(73, 73, 255);
+}
+
+atomic-color-facet[field='cat_color']::part(value-Multi) {
+  background-image: linear-gradient(
+    315deg,
+    #b43a3a 0%,
+    #fffb71 16%,
+    rgba(137, 255, 126) 33%,
+    #2de9e1 50%,
+    #5f6cdb 66%,
+    #d79fdb 83%,
+    #ff6464 100%
+  );
+  border: 1px solid var(--atomic-neutral);
+}
+:root {
+  --spacing: 16px;
+  --spacing-col: 32px;
+  --search-box-width: 648px;
+}
+
+atomic-search-box {
+  margin-top: 1rem;
+}
+
+atomic-search-interface {
```

Comment:
What's the support status of layouts for Atomic React ?

Reply:
Should work... I could do the change in this PR 👍